src/plugins/console/server/lib/proxy_request.ts

```diff
@@ -58,11 +54,18 @@ export const proxyRequest = ({
   timeout,
   payload,
   rejectUnauthorized,
+  path,
```

Comment:
It is not immediately clear to me what the difference between `path` and `pathname` are. Can you use more specific names to show how they are different?